packages/react-vapor/src/components/textInput/useForm.ts

```diff
@@ -0,0 +1,12 @@
+import {useContext, useMemo} from 'react';
+import {TextInputState} from './TextInputReducer';
+import {FormComponent, FormContext} from '../form/FormProvider';
+
+export const useForm = (form: FormComponent): {state: Record<string, TextInputState>} => {
+    const formContext = useContext(FormContext);
+    if (formContext === undefined) {
+        throw new Error('useForm must be used within a FormProvider.');
+    }
+    const state = formContext.state[form] || {};
+    return useMemo(() => ({state}), [state]);
```

Comment:
I think it should provide a `dispatch` function just like the `useTextInput`. I would see myself using it to clear the value of multiple text inputs at once for example.